index.js

```diff
@@ -81,6 +116,11 @@ async function middleware (params, path) {
           if (/\/send\s\w+\s\w+\s\w+$/.test(message.text)) {
             const [_, text, gid, tid] = message.text.split(' ');
             clients[path].pushV2(gid, tid, createResponse(responseMap, text));
+          } else if (/\/searchWorkflow\s\w+$/.test(message.text)) {
+            const [_, keyword] = message.text.split(' ');
+            const userId = event.source.userId;
+
+            searchWorkflow(clients[path], replyToken, userId, keyword)
```

Comment:
Let's await the search function here to get the workflows and put the reply code after the function completes.

Reply:
Done! You can have a look at the new commit.

I had to add a try/catch to be sure that we catch the error from axios.